membersrvc/ca/ca.go

```diff
@@ -34,7 +34,6 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3" // TODO: justify this blank import or remove
```

Comment:
This line is equired to successfully initialized the sqlite3 driver otherwise the golang db infrastructure will not the able to find the driver. Look at this example https://astaxie.gitbooks.io/build-web-application-with-golang/content/en/05.3.html